Sources/SourceKit/sourcekitd/SwiftLanguageServer.swift

```diff
@@ -758,4 +840,39 @@ extension sourcekitd_uid_t {
         return nil
     }
   }
+
+  func asSymbolKind(_ vals: sourcekitd_values) -> SymbolKind? {
+    switch self {
+      case vals.decl_class:
+        return .class
+      case vals.decl_function_method_instance,
```

Comment:
How about

```
case vals.decl_function_constructor:
  return .constructor
```

Reply:
I'm not sure how would I go about this, because sourcekit seems to return "source.lang.swift.decl.function.method.instance" as the type for both inits and normal methods

Reply:
Weird, okay we can skip this for now then.

Reply:
I just found this discussion. since `init` and `deinit` is a keyword, maybe we should check method name and apply `.keyword` here, however `.constructor` is semantially more correct)